src/main/java/org/jabref/gui/icon/JabRefIconView.java

```diff
@@ -2,35 +2,49 @@
 
 import javafx.beans.property.ObjectProperty;
 import javafx.beans.property.SimpleObjectProperty;
+import javafx.css.Size;
+import javafx.css.SizeUnits;
+
+import org.jabref.gui.icon.IconTheme.JabRefIcons;
 
 import com.tobiasdiez.easybind.EasyBind;
-import de.jensd.fx.glyphs.GlyphIcon;
+import org.kordamp.ikonli.javafx.FontIcon;
 
-public class JabRefIconView extends GlyphIcon<IconTheme.JabRefIcons> {
+public class JabRefIconView extends FontIcon {
 
     /**
      * This property is only needed to get proper IDE support in FXML files
      * (e.g. validation that parameter passed to "icon" is indeed of type {@link IconTheme.JabRefIcons}).
      */
     private final ObjectProperty<IconTheme.JabRefIcons> glyph;
+    private final ObjectProperty<Number> glyphSize;
 
-    public JabRefIconView(IconTheme.JabRefIcons icon, String iconSize) {
-        super(IconTheme.JabRefIcons.class);
+    public JabRefIconView(JabRefIcons icon, int size) {
+        super(icon.getIkon());
         this.glyph = new SimpleObjectProperty<>(icon);
-        EasyBind.subscribe(glyph, this::setIcon);
-        setIcon(icon);
-        setStyle(String.format("-fx-font-family: %s; -fx-font-size: %s;", icon.fontFamily(), iconSize));
+        this.glyphSize = new SimpleObjectProperty<>(size);
+
+        EasyBind.subscribe(glyph, glyph -> setIconCode(glyph.getIkon()));
+        EasyBind.subscribe(glyphSize, glyphsize -> setIconSize(glyphsize.intValue()));
     }
 
     public JabRefIconView(IconTheme.JabRefIcons icon) {
-        this(icon, "1em");
+
+        super(icon.getIkon());
```

Comment:
Refactor to `this(icon, calculateDefaultSize())` ?

Reply:
Doesn't work because I need getFont from the super class.